content/guides/tanzu-labs-engineer-faq/index.md

```diff
@@ -0,0 +1,364 @@
+
+That hyper-productivity caused by pairing comes at a cost: it's exhausting. And if you stay in that state too long, you'll start making bad decisions. [Science indicates](https://hbr.org/2017/05/your-brain-can-only-take-so-much-focus) that the human brain works best when it gets to alternate between states of focus and distraction. That's why pairing culture requires a culture of taking breaks. Every hour or so, the pairs need to stand up and do something completely different. Zone out, listen to music, play a game, etc.
+
+Pairing also means we can spend less time on asynchronous code reviews. On teams that don't pair, developers spend a good portion of their time reading pull-requested code and writing comments. Then the original author of that code has to respond to those comments, revise their code, and resubmit it. The extra overhead of code review and code revision can easily take up ~50% of a team's working hours, so by substituting pairing for code review we lose nothing. In fact, we save time, because the real-time code review provided by pairing is more efficient than typing comments back and forth on Github.
+
+In answer to the question "So I have to hire twice as many engineers to get the same amount of work done?": in an ideal scenario, you won't have to hire more engineers. Those four engineers will be more productive paired up than when they were solo. However, reality isn't ideal. Many companies may still need to hire more engineers because if they have a traditional IT culture, it's likely they have a fair number of engineers in their current workforce who will refuse pair. To achieve the level of effectiveness we expect from pairing, each person needs to value others' perspectives as much as their own, and be willing to be vulnerable, exposing what they don't know. Not everyone is willing to develop the empathy and relationship skills required to collaborate at this level. Pairing also exposes intellectual insecurities. To pair, you have to overcome your own ego – you have to be willing to make mistakes in front of your pair. Some engineers are terrified of this prospect, and would rather quit than face their fears.
```

Comment:
```suggestion
In answer to the question "So I have to hire twice as many engineers to get the same amount of work done?": in an ideal scenario, you won't have to hire more engineers. Those four engineers will be more productive paired up than when they were solo. However, reality isn't ideal. Many companies may still need to hire more engineers because if they have a traditional IT culture, it's likely they have a fair number of engineers in their current workforce who will refuse to pair. To achieve the level of effectiveness we expect from pairing, each person needs to value others' perspectives as much as their own, and be willing to be vulnerable, exposing what they don't know. Not everyone is willing to develop the empathy and relationship skills required to collaborate at this level. Pairing also exposes intellectual insecurities. To pair, you have to overcome your own ego – you have to be willing to make mistakes in front of your pair. Some engineers are terrified of this prospect, and would rather quit than face their fears.
```